docs/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+## Is this UserStory done?
+
+You might be confused about what it really means for a User Story to be 'done'.
```

Comment:
This this belong in every pull request? :D

Anyway, I would also add a section to remind people to link their issues. 

```
## Related Issues

- Closes #
- Fixes #
```